packages/cli/package.json

```diff
@@ -47,6 +47,7 @@
     "@code-pushup/utils": "0.56.0",
     "yargs": "^17.7.2",
     "ansis": "^3.3.0",
+    "tslib": "^2.6.2",
```

Comment:
Why does `tslib` need to be added as a production dependency everywhere? :raised_eyebrow:

Reply:
It seems to be fixed by setting `"importHelpers": false` in `tsconfig.base.json` as we do not need import helpers anyway.